README.md

```diff
@@ -16,6 +18,8 @@ Overview
 
 How do I use Tableau Log Viewer?
 ---------------
+You can get the latest release in the [Releases Section](https://github.com/tableau/tableau-log-viewer/releases)

```

Comment:
Good to see this added, as we had a couple questions around this. Could you also add a few lines about the "installation"? Some thing along the lines:
Download the corresponding Windows or Mac/OSX zip file and unzip it to a location.